consul_exporter.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"flag"
+	_ "fmt"
```

Comment:
I started from the haproxy_exporter importers, then just started commenting things out to see what I needed. The missing step was going back to actually delete the ones I didn't.